health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -279,14 +371,14 @@ const ViewHierarchy = () => {
                                             <div>
                                                 <div style={{fontWeight:"600", fontSize:"1.2rem"}}>
                                                     {/* {t(hierItem?.boundaryType)} */}
-                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase())}`}
+                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toLowerCase().replace(/\s+/g, "_"))}`}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)